### PR TITLE
Add a Go package for exporting a urfave/cli command

### PR DESCRIPTION
**Pull request by bcmendoza**
`config/config.go` exports a urfave/cli command, `KubernetesCommand`, which enables passing flags for templating manifests which are written to stdout.

This PR also adds a more organized `config` directory structure and simplifies resource names.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	DockerConfigBase64 string
+}
+
+func loadManifests(dirPath string, conf manifestConfig) error {
+	if conf.DockerUsername == "" || conf.DockerPassword == "" {
+		return fmt.Errorf("missing docker credentials")
+	}
+	conf.DockerConfigBase64 = genDockerConfigBase64(conf.DockerUsername, conf.DockerPassword)
+	if conf.DockerImageURL == "" {
+		conf.DockerImageURL = "docker.greymatter.io/development/gm-operator:0.0.1"
```
Comment (dontlaugh) [inline config/config.go]: make this image a global, private constant

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+	}
+
+	return string(yml), nil
+}
+
+func mkKyamlFileSys(efs embed.FS) (filesys.FileSystem, error) {
+	kfs := filesys.MakeFsInMemory()
+	loadFunc := mkFileLoader(efs, kfs)
+	if err := fs.WalkDir(efs, ".", loadFunc); err != nil {
+		return kfs, err
```
Comment (dontlaugh) [inline config/config.go:129]: should this be `return nil, err`? or a zero-value'd struct literal, if kfs isn't an interface.

File: config/config.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+	}
+	return kfs, nil
+}
+
+func mkFileLoader(efs embed.FS, kfs filesys.FileSystem) func(string, fs.DirEntry, error) error {
```
Comment (dontlaugh) [inline config/config.go:134]: Let's add a docstring here.

File: config/config_test.go [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/urfave/cli/v2"
+)
+
```
Comment (dontlaugh) [inline config/config_test.go:12]: Very nice tests :+1:

**Review by moosecanswim — DISMISSED** [review]
LGTM

**Review by dontlaugh — CHANGES_REQUESTED** [review]
A few changes, mostly stylistic

**Review by dontlaugh — APPROVED** [review]
Looks good. Go ahead and merge this, if it makes creating the corresponding PR in the cli project easier.

**Review by dontlaugh — APPROVED** [review]

**Pull request merged**